### PR TITLE
Fixed react warnings in StoryPromo

### LINKED ISSUE #3449: Fix React warnings in src/app/containers/StoryPromo/index.test.jsx
**Describe the bug**
There are React warnings in unit tests.

**To Reproduce**
Go to simorgh in terminal
Run npm run test:unit
Observe React warnings:
```
  console.error src/testHelpers/setupTests.js:53
    Warning: Failed prop type: Invalid prop `item` supplied to `StoryPromo`.
        in StoryPromo (created by WrappedStoryPromo)
        in WrappedStoryPromo

  console.error src/testHelpers/setupTests.js:53
    Warning: Failed prop type: Invalid prop `item` of type `array` supplied to `LinkContents`, expected `object`.
        in LinkContents (created by StoryPromo)
        in StoryPromo (created by WrappedStoryPromo)
        in RequestContextProvider (created by WrappedStoryPromo)
        in ServiceContextProvider (created by WrappedStoryPromo)
        in WrappedStoryPromo
```
**Expected behaviour**

No React warning for test suite.

**Testing notes**

No testing required.

### PR DESCRIPTION
**Pull request by hotinglok**
Resolves #3449 

**Overall change:** 
Fixed react warnings.

**Code changes:**
- Changed item to correct prop `indexAlsosItem.relatedItems`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/StoryPromo/index.test.jsx [hunk 1]
```diff
@@ -419,7 +419,7 @@ describe('StoryPromo Container', () => {
 
       it('should render a related item not contained within a list', () => {
         const { container } = render(
-          <WrappedStoryPromo item={[indexAlsosItem[0]]} topStory />,
+          <WrappedStoryPromo item={indexAlsosItem.relatedItems[0]} topStory />,
```
Comment (dr3) [inline src/app/containers/StoryPromo/index.test.jsx]: Im not sure if this change is right, need to confirm with WS fp pod what this test is trying to do

Comment (dr3) [reply src/app/containers/StoryPromo/index.test.jsx]: Should it be this?

Is it testing if `relatedItems` isnt an array?

```jsx
const indexAlsosItemRelatedNoList = indexAlsosItem;
indexAlsosItemRelatedNoList.relatedItems = indexAlsosItemRelatedNoList.relatedItems[0];

const { container } = render(
    <WrappedStoryPromo item={indexAlsosItemRelatedNoList} topStory />,
);
```

This causes `Invalid prop 'alsoItems' of type 'object' supplied to 'IndexAlsosContainer', expected an array.` but its all I can think makes sense for `related item not contained within a list`

Comment (andrew-nowak) [inline src/app/containers/StoryPromo/index.test.jsx]: add a line before like

`const onlyOneRelatedItem = { ...indexAlsosItem, relatedItems: [ indexAlsosItem.relatedItems[0] ] };`
(maybe way up near where all the other fixtures are defined)

then

```suggestion
          <WrappedStoryPromo item={onlyOneRelatedItem} topStory />,
```

cc @DenisHdz for double checking

Comment (dr3) [reply src/app/containers/StoryPromo/index.test.jsx]: Ohhhhh `not contained within a list` means, the array has length 1 so doesnt include a `li` and `ul` elements, damn my FP knowledge is getting rusty haha

**Review by andrew-nowak — APPROVED** [review]
looks good

**Review by jroebu14 — APPROVED** [review]

**Pull request merged**